sunpy/coordinates/frames.py

```diff
@@ -23,13 +23,31 @@
 
 class SunPyBaseCoordinateFrame(BaseCoordinateFrame):
     """
-    Inject a nice way of representing the object which the coordinate represents.
+    * Defines a default longitude wrap angle of 180 degrees, which can be overridden by the class
+      variable `_wrap_angle`.
+    * Inject a nice way of representing the object which the coordinate represents.
     """
+    _wrap_angle = 180*u.deg
 
     def __init__(self, *args, **kwargs):
         self.object_name = None
+
+        # If wrap_longitude=False is passed in, do not impose a specific wrap angle for the frame
+        if not kwargs.pop('wrap_longitude', True):
+            self._wrap_angle = None
+
         return super().__init__(*args, **kwargs)
 
+    def represent_as(self, base, s='base', in_frame_units=False):
+        """
+        If a frame wrap angle is set, use that wrap angle for any spherical representations.
+        """
+        data = super().represent_as(base, s, in_frame_units=in_frame_units)
+        if self._wrap_angle is not None and \
```

Comment:
line break using `()` rather than `\`?

Reply:
I originally had it that way, but when using the `(`, the way that I want to indent the following line becomes exactly 4 spaces, so it's easy to confuse for an actual indention (and the bot complained as well).  So, I changed it to the `\` so that I could indent the folllowing line only 3 spaces and break that confusion.  It's admittedly silly.  How would you indent it?

Reply:
Fair enough. I just have a pathological hatred for `\` lol